<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">resizeMode: true;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>0</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
    color: black;
    border: 2px solid #555555;
background-color: #ffffff; /* Green */
    border: none;
margin: 4px 4px;
border-radius: 5%;
    color: white;
    padding: 10px 10px;
    text-align: center;
    text-decoration: none;
    display: inline-block;
    font-size: 14px;
    cursor: pointer;
color: hsla(#000000);
</string>
    </property>
    <property name="text">
     <string>Черный</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
    color: black;
    border: 2px solid #555555;
background-color: #ffffff; /* Green */
    border: none;
margin: 4px 4px;
border-radius: 5%;
    color: white;
    padding: 10px 10px;
    text-align: center;
    text-decoration: none;
    display: inline-block;
    font-size: 14px;
    cursor: pointer;
color: hsla(#000000);
</string>
    </property>
    <property name="text">
     <string>Белый</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>190</y>
      <width>261</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>Авторизация</string>
    </property>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>160</y>
       <width>81</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #000000</string>
     </property>
     <property name="text">
      <string>Log in</string>
     </property>
    </widget>
    <widget class="QLabel" name="error">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>130</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 12px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> border: 2px solid #555555;
    border: none;
margin: 4px 4px;
border-radius: 5%;
    padding: 10px 10px;
    text-align: center;
    text-decoration: none;
    display: inline-block;
    font-size: 14px;
    cursor: pointer;</string>
     </property>
     <property name="placeholderText">
      <string>Login</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="passworld">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> border: 2px solid #555555;
    border: none;
margin: 4px 4px;
border-radius: 5%;
    padding: 10px 10px;
    text-align: center;
    text-decoration: none;
    display: inline-block;
    font-size: 14px;
    cursor: pointer;</string>
     </property>
     <property name="placeholderText">
      <string>Passworld</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>408</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
